demo/lightning/text-based/distributed/requirements.txt

```diff
@@ -0,0 +1,6 @@
+torch
+dataflux
```

Comment:
Is this supposed to be here? I think gcs-torch-dataflux is the right one isn't it?